src/adapters/PeopleAdapter.test.js

```diff
@@ -0,0 +1,28 @@
+import {isObservable} from 'rxjs';
```

Comment:
I prefer to have `rxjs` globally available in our all testing module. but we can do that later.